etc/init.d/zfs-zed.in

```diff
@@ -45,8 +39,7 @@ ZED_PIDFILE="@runstatedir@/$ZED_NAME.pid"
 
 do_depend()
 {
-	before zfs-import
-	after sysfs
+	after sysfs zfs-mount
```

Comment:
@MrStaticVoid My line of thinking when I asked that question was not one of my finer moments. The best thing here would be to specify `after zfs-mount localmount` in the zfs-zed.in script. The ordering of zfs-mount and localmount will depend on whether / is on ZFS or not, so this tackles both.